characters-attack.html

```diff
@@ -1,44 +1,36 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="pl">
 
 <head>
     <meta charset="utf-8">
     <title>OVERWATCH - POSTACIE NATARCIA</title>
-    <meta name="keywords" content="owerwatch, owerwatch postacie, gry strzelanie" />
+    <meta name="keywords" content="overwatch, overwatch heroes, game shooting" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"  />
     <link rel="stylesheet" href="style.css" type="text/css" />
-
 </head>
 
 <body>
-    <div id="container">
-        <div id="logo">
+    <section class="container">
+        <div id="top-title">
             OVERWATCH - POSTACIE NATARCIA
         </div>
-
-        <div id="menuT"> 
-
-            <div style="margin-left: 75px;" class="option" >START</div>      
-            <div class="option" >POSTACIE NATARCIA</div>
-            <div class="option" >POSTACIE TANK</div> 
-            <div style="margin-right: 75px;"  class="option" >POSTACIE WSPARCIA</div>
-            <div style="clear: both;" ></div>
-
-
-        </div>  
-        
-        <div class="descripe">
-            <div id="ashe">
-                <div class="imie">
-                    ASHE - Elizabeth Caledonia
-                </div>
-
-                 <div class="obraz">
-                     <img src="postacie/natarcie/ashe.jpg">
+        <nav>                 
+            <div class="option with-margin"><a style="color:white" href="index.html">START </a></div>      
+            <div class="option with-margin"><a style="color: white;" href="characters-attack.html">POSTACIE NATARCIA </a> </div>
+            <div class="option with-margin"><a style="color: white;" href="characters-tank.html">POSTACIE TANK</a></div> 
+            <div class="option with-margin"><a style="color: white;" href="characters-healer.html">POSTACIE WSPARCIA</a></div>
+            <div style="clear: both;"></div>
```

Comment:
as `nav` has `display: flex`, this div shouldn't be needed anymore. So - simply delete it